Library/Homebrew/cask/cask.rb

```diff
@@ -175,6 +175,8 @@ def to_h
         "url"            => url,
         "appcast"        => appcast,
         "version"        => version,
+        "installed"      => versions,
```

Comment:
This should be a boolean.

Reply:
I was trying to match the keys with [formula.rb](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/formula.rb#L1849https://github.com/Homebrew/brew/blob/master/Library/Homebrew/formula.rb#L1849) which lists installed kegs with the `installed` key. I see why it makes sense as a boolean though. How about changing it to a boolean, and then adding `"installed_versions" => versions` to provide additional details?

Reply:
> How about changing it to a boolean, and then adding `"installed_versions" => versions` to provide additional details?

You can’t have more than one version of a cask installed at once—that would make no sense for most software. So just the boolean, and no further details necessary.

Reply:
```suggestion
        "installed"      => versions.last,
```

Reply:
```suggestion
        "installed"      => versions.last,
```